### PR TITLE
Refactors UnitsViewController

### PR DESCRIPTION
**Pull request by philium**
Refactors UnitsViewController to use modern APIs and be more streamlined.

### CODE REVIEW

File: Toolkit/ArcGISToolkit/MeasureToolbar.swift [hunk 1]
```diff
@@ -266,7 +278,9 @@ public class MeasureToolbar: UIView, AGSGeoViewTouchDelegate {
         segControl.setImage(measureFeatureImage, forSegmentAt: 2)
         segControlItem = UIBarButtonItem(customView: segControl)
         
-        resultView.buttonTapHandler = unitsButtonTap
```
Comment (philium) [inline Toolkit/ArcGISToolkit/MeasureToolbar.swift]: This was giving the result view a strong reference to the measure toolbar, thereby creating a retain cycle and preventing the toolbar from being deallocated.

File: Toolkit/ArcGISToolkit/MeasureToolbar.swift [hunk 2]
```diff
@@ -297,30 +311,14 @@ public class MeasureToolbar: UIView, AGSGeoViewTouchDelegate {
         
         // notification
         
-        NotificationCenter.default.addObserver(forName: NSNotification.Name.AGSSketchEditorGeometryDidChange, object: lineSketchEditor, queue: nil, using: sketchDidChange)
```
Comment (philium) [inline Toolkit/ArcGISToolkit/MeasureToolbar.swift]: This (and the subsequent line) were also preventing the measure toolbar from being deallocated.

File: Toolkit/ArcGISToolkit/UnitsViewController.swift [hunk 3]
```diff
@@ -125,155 +110,104 @@ public class UnitsViewController: TableViewController, UINavigationBarDelegate,
...
-        }
-    }
-    
-    public func position(for bar: UIBarPositioning) -> UIBarPosition {
-        return .topAttached
     }
     
     // MARK: TableView delegate/datasource methods
     
-    func tableView(_ tableView: UITableView, didSelectRowAtIndexPath indexPath: IndexPath) {
```
Comment (philium) [inline Toolkit/ArcGISToolkit/UnitsViewController.swift]: This method had the wrong signature, so it was never getting called. Selecting a unit had not effect.

**Comment by rolson** [comment]
Thanks @philium, @mhdostal can you please review?

**Pull request merged**